app/pages/sections/service-recipients/manifest.json

```diff
@@ -7,7 +7,16 @@
     "select": "Select all",
     "deselect": "Deselect all"
   },
-  "organisationHeading": "Organisation",
-  "odsCodeHeading": "ODS Code",
+  "serviceRecipientsTable": {
+    "columnInfo": [
+      {
+        "data": "Organisation"
+      },
+      {
+        "data": "ODS Code"
+      }
+    ],
```

Comment:
I think once you add a cellInfo this will tidy up the contextCreator a fair bit. Have a look at how we have done it for the `catalogue-solution/order-item/flat/declarative/manifest.json`